MaxText/configs/inference.yml

```diff
@@ -0,0 +1,626 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sentinel is a reminder to choose a real run name.
+# If there is already a checkpoint under this run, that checkpoint will auto-resume.
+run_name: ""
```

Comment:
Sorry, I made this comment in the group chat, but forget to comment here.

Instead of duplicating the whole yml, is that possible to just override the delta like this yml file? https://github.com/AI-Hypercomputer/maxtext/blob/main/MaxText/configs/dpo.yml So, we only need to maintain the delta in this file. I think it's inference team's call, but for code quality, I would prefer just have delta in this file.

cc @vipannalla @mailvijayasingh